ui/src/pages/home/apps-section/app-grid/app-grid.tsx

```diff
@@ -11,16 +11,11 @@ export const AppGrid = ({ apps }: AppsGridProps): React.ReactElement => {
     <>
       {apps.map((app: JhApp, index: number) => (
         <AppCard
-          id={app.id}
+          {...app}
           key={`app-${app.id}-${index}`}
           title={app.name}
-          description={app.description}
-          thumbnail={app.thumbnail}
-          framework={app.framework}
-          url={app.url}
           serverStatus={app.status}
           lastModified={app.last_activity}
-          username={app.username}
           isPublic={app.public}
           isShared={app.shared}
```

Comment:
Could these props be renamed to simplify even further, i.e. title becomes name, serverStatus changed to status, isPublic to public

Reply:
Some could be, but 1 in particular cannot, `public`, as it has meaning in TypeScript.

Reply:
And I try to stick with camel casing for variables and props, whereas some of the values coming from the API are snake cased.

Reply:
Ok. Based on that. Closing this.